### DISCUSSION TITLE
Include private posts in liveblog

### DISCUSSION THREAD

**Discussion by feministclickback** [jeffreyvr/easy-liveblogs#16] [Ideas]
Hi, great plugin!
Question - is there a filter or something like it that I can use to include private posts in the liveblog? My editors would like to have a preview of what their entry would look like live before they publish. 
(If filter doesn't exist, I think this would be a very useful new feature)
Thanks!
Tina

**Comment by jeffreyvr** [comment]
Thanks! There is no filter for it yet, but it could be added here:
https://github.com/jeffreyvr/easy-liveblogs/blob/master/includes/api/class-elb-feed-factory.php#L48

Probably should be named something like `elb_api_get_entries_args`. 

If you're up for it, feel free to create a PR. Otherwise I'll add it at a later moment when I have time.

**Reply by feministclickback** [reply]
Thanks for the quick reply! I'll see later this week if I can do it :)

**Reply by jeffreyvr** [reply]
The filter is available now in version 2.3.4.
